roles/common/tasks/main.yml

```diff
@@ -67,19 +67,29 @@
   command: hostnamectl set-hostname --static --transient {{ inventory_hostname }}
   tags: dns
 
-# setup base ssh and reject rules
-- name: setup ssh iptables rule
-  iptables:
-    port: 22
-    protocol: tcp 
-    comment: "ssh incoming"
+# setup base ssh rules
+- name: turn off firewalld
+  service:
+    name: firewalld
```

Comment:
Do you want the removal from openstack-common to be in this PR or a separate one?